boards/pinetime/include/gpio_params.h

```diff
@@ -0,0 +1,38 @@
+/*
```

Comment:
The file is empty (e.g no saul gpio), why not just remove it (for now) ?

Reply:
removed